docker/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_DIR="$(realpath $(dirname $BASH_SOURCE[0]))"
+
+cd "$SCRIPT_DIR"/wine-docker
+docker build --platform linux/amd64 -t wine-docker:devel --build-arg WINE_FLAVOUR=devel .
```

Comment:
Consider naming this image something more unique to avoid the potential for collisions.